Lib/typing.py

```diff
@@ -2847,10 +2865,10 @@ def _make_eager_annotate(types):
                      for key, val in types.items()}
     def annotate(format):
         match format:
```

Comment:
similarly here, I think we should probably just get `annotationlib` once at the beginning of this `annotate` function and store it as a local variable rather than repeatedly loading it from the `_lazy_annotationlib` object multiple times in this function